torchaudio/transforms.py

```diff
@@ -996,6 +996,11 @@ class Fade(torch.nn.Module):
         fade_out_len (int, optional): Length of fade-out (time frames). (Default: ``0``)
         fade_shape (str, optional): Shape of fade. Must be one of: "quarter_sine",
             "half_sine", "linear", "logarithmic", "exponential". (Default: ``"linear"``)
+            
+     Example
+        >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
+        >>> transform = transforms.Fade(fade_in_len=1, fade_out_len=2, fade_shape='linear')
```

Comment:
1 and 2 sample points seem too few. Could you change it to 1 second and 2 seconds? 
```suggestion
        >>> transform = transforms.Fade(fade_in_len=sample_rate, fade_out_len=2 * sample_rate, fade_shape='linear')
```

Reply:
Surely!